<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Charity</class>
 <widget class="QWidget" name="Charity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1776</width>
    <height>1122</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>HP Simplified</family>
    <pointsize>46</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableViewCharity">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>260</y>
     <width>471</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLogout">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>260</y>
     <width>150</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kirjaudu ulos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonReturnMenu">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>840</y>
     <width>150</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Päävalikko</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDonate20">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>670</y>
     <width>150</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>20€</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCharityName">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>520</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDonate5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>670</y>
     <width>150</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5€</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDonate10">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>670</y>
     <width>150</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>520</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Lahjoituskohteesi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>210</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Valitse lahjoituskohde</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>600</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Lahjoitus summa</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCharityAmount">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>600</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDonateComplete">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>840</y>
     <width>481</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lahjoita tästä</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCharity">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>800</width>
     <height>160</height>
    </rect>
   </property>
   <property name="text">
    <string>Lahjoitus</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelErrorText">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>800</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ErrorText</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
